src/collection-browser.ts

```diff
@@ -368,24 +385,48 @@ export class CollectionBrowser
     const sortField = SortFieldToMetadataField[this.selectedSort];
     if (!sortField) return;
     this.sortParam = { field: sortField, direction: this.sortDirection };
+
+    this.analyticsHandler?.sendEventNoSampling({
+      category: this.analyticsCategories.default,
+      action: this.analyticsActions.sortBy,
+      label: `${sortField} - ${this.sortDirection}`,
+    });
   }
 
   private displayModeChanged(
     e: CustomEvent<{ displayMode: CollectionDisplayMode }>
   ) {
     this.displayMode = e.detail.displayMode;
+
+    this.analyticsHandler?.sendEventNoSampling({
+      category: this.analyticsCategories.default,
+      action: this.analyticsActions.displayMode,
+      label: this.displayMode,
+    });
   }
 
   private selectedTitleLetterChanged() {
     this.titleQuery = this.selectedTitleFilter
       ? `firstTitle:${this.selectedTitleFilter}`
       : undefined;
+
+    this.analyticsHandler?.sendEventNoSampling({
+      category: this.analyticsCategories.default,
+      action: this.analyticsActions.sortByTitle,
+      label: `${this.titleQuery}`,
```

Comment:
```suggestion
      label: this.titleQuery ?? 'unset',
```
sometimes the label values have string interpolation, some none...please do take a sweep to check strip unnecessary transforms, thanks ^_^